.github/ISSUE_TEMPLATE/RFC.md

```diff
@@ -0,0 +1,71 @@
+---
+name: RFC
+about: Propose new feature(s) or major changes to the component
+title: "[RFC]: [TITLE OF RFC]"
```

Comment:
Maybe we want to get rid of the `[RFC]` prefix here aswell as we did for BC break?